### ISSUE TITLE
Add a script to download all HTML files

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#230]
For #221 we need a way to download all HTML sim files on the website.  After doing that *and* after S2015R sims are all redeployed, we should use it to generate a new report.

**Comment by samreid** [comment]
Unassigning this since S2015R completion is quite a ways away.

**Comment by pixelzoom** [comment]
Do you mean downloading the latest version of each html sim, as identified by `RewriteRule latest` in the sim's .htaccess file?

**Comment by samreid** [comment]
Yes, and I was planning to discuss with @jonathanolson and @aaronsamuel137 to find the best way to enumerate all of the links.

**Comment by pixelzoom** [comment]
Brainstorming, from most to least desirable:

• Write a web service that allows the script to ask for this info.
 
• Scrape the URLs from some page on the website, e.g. - Simulations > By Device > iPad/Table? (http://phet.colorado.edu/en/simulations/category/by-device/ipad-tablet)

• Read the .htaccess files from the serve, assemble the URLs in the script. (Yuck.)

**Comment by jonathanolson** [comment]
> Write a web service that allows the script to ask for this info.

If not already around, a comprehensive service for this would be great (rather than the somewhat-lacking project-sorted-simulations.csv).

**Comment by aaronsamuel137** [comment]
I think writing a web service is a good idea, and probably wouldn't be too difficult. Feel free to assign to me for this part of the issue.

**Comment by samreid** [comment]
A web service for this is probably the next step, but won't be of much use until several S2015R deploys have been made.
